climakitae/core/data_export.py

```diff
@@ -689,13 +694,11 @@ def _export_to_csv(data: xr.DataArray | xr.Dataset, save_name: str):
     print("Exporting specified data to CSV...")
     _warn_large_export(est_file_size, 1.0)
 
-    ftype = type(data)
-
-    if ftype == xr.core.dataarray.DataArray:
-        df = _dataarray_to_dataframe(data)
-
-    elif ftype == xr.core.dataset.Dataset:
-        df = _dataset_to_dataframe(data)
+    match type(data):
+        case xr.core.dataarray.DataArray:
+            df = _dataarray_to_dataframe(data)
+        case xr.core.dataset.Dataset:
+            df = _dataset_to_dataframe(data)
```

Comment:
This formatting needs to be changed, and a final case needs to be added

Reply:
Fixed in [ce4dd86](https://github.com/cal-adapt/climakitae/pull/564/commits/ce4dd86a4fbc5b25eb8a5ee8b83cbdfc0de3a764)